demos/smartlab_demo/python/README.md

```diff
@@ -0,0 +1,85 @@
+Action recognition architecure uses two encoders for front-view and top-view respectively, and a single decoder.
+Object detection uses two models for each view to detect large object and small objects, respectively.
+The following pre-trained models are delivered with the product:
+
+
+* `smartlab-action-recognition-encoder-0001` + `smartlab-action-recognition-decoder-0001`, which are models for identifying actions 2 actions of smartlab (adjust_rider, put_take).
+
+* `i3d-rgb-tf` + `smartlab-sequence-modelling-0001`, which are other models for identifying actions 2 actions of smartlab (adjust_rider, put_take).
+
+* `smartlab-object-detection-0001` + `smartlab-object-detection-0002` + `smartlab-object-detection-0003` + `smartlab-object-detection-0004`, which are models for detecting smartlab objects (10 objects). They detect balance, weights, tweezers, box, battery, tray, ruler, rider, scale, hand.
+
+## How It works
+
+The demo pipeline consists of several steps:
+
+* `Decode` reads frames from the input videos
+* `Detector` detects smartlab objects (balance, weights, tweezers, box, battery, tray, ruler, rider scale, hand)
+* `Segmentor` segments video frames based on action of the frame
+* `Evaluator` calculates scores of the current state
+* `Display` displays detected objects, recognized action, calculated scores on the current frame
+
+
+> **NOTE**: By default, Open Model Zoo demos expect input with BGR channels order. If you trained your model to work with RGB order, you need to manually rearrange the default channels order in the demo application or reconvert your model using the Model Optimizer tool with the `--reverse_input_channels` argument specified. For more information about the argument, refer to **When to Reverse Input Channels** section of [Converting a Model Using General Conversion Parameters](https://docs.openvino.ai/latest/openvino_docs_MO_DG_prepare_model_convert_model_Converting_Model.html#general-conversion-parameters).
+
+## Preparing to Run
+For demo input image or video files, you need to provide smartlab videos (https://storage.openvinotoolkit.org/data/test_data/videos/smartlab/).
+The list of models supported by the demo is in `<omz_dir>/demos/smartlab_demo/python/models.lst` file.
+This file can be used as a parameter for [Model Downloader](../../../tools/model_tools/README.md) and Converter to download and, if necessary, convert models to OpenVINO format (\*.xml + \*.bin).
+
+An example of using the Model Downloader:
+
+```sh
+omz_downloader --list models.lst
+```
+
+An example of using the Model Converter:
+
+```sh
+omz_converter --list models.lst
+```
+
+### Supported Models
+* smartlab-action-recognition-encoder-0001
+* smartlab-action-recognition-decoder-0001
+* smartlab-object-detection-0001
+* smartlab-object-detection-0002
+* smartlab-object-detection-0003
+* smartlab-object-detection-0004
+* smartlab-sequence-modelling-0001
+* i3d-rgb-tf
```

Comment:
Please add another test with `smartlab-sequence-modelling-0001` and `i3d-rgb-tf`